hybrid/layout/pv_design_utils.py

```diff
@@ -272,14 +299,14 @@ def get_module_attribs(pvsam_model: pv.Pvsamv1) -> dict:
             'length':       module_length,
             'I_mp_ref':     I_mp,
             'I_sc_ref':     I_sc,
-            'P_mp_ref':     P_mp,
+            'P_mp_ref':     P_mp * 1e-3,
```

Comment:
could you add units to these key-value pairs too as you did below?

Reply:
Done